README.md

```diff
@@ -396,29 +396,33 @@ Here is a summary of the commands available in-game. All commands are prefixed b
 </tbody>
 </table>
 
-## Credits
-### Contributors to DLUv3
-* DarwinAnim8or
-* Wincent01
-* Mick
-* averysumner (codeshaunted)
-* Jon002
-* Jonny
-* Xiphoseer
+## Current DLU Team
```

Comment:
@Jettford Ideally I'd like the Darkflame Universe team specifically still called out here; calling them just "Previous Maintainers / Developers" isn't ideal, imo. They deserve a distinction.

Reply:
> @Jettford Ideally I'd like the Darkflame Universe team specifically still called out here; calling them just "Previous Maintainers / Developers" isn't ideal, imo. They deserve a distinction.

Agreed.

Reply:
I have updated it to something simpler and closer to what it was before I touched it. Please let me know if you have any other suggestions.

Reply:
@Jettford can you please put it back to how you had it before, just change the pre-FOSS contributors to “Original DLU Team” or something along those lines?

Reply:
Only other suggestion is to still rename the "Contributors to DLU Pre Launch" to "Darkflame Universe Team, Past and Present"

Reply:
Alright, I have written up my thoughts in the DLU Developers Group discord and am awaiting thoughts of the rest of the DLU team before I make anymore changes.

Reply:
You were given no promise of being a part of Darkflame Universe when contributing to this project. You contribute to an open source project as an individual and if we ever recruit more to our team or do anything else as a group, we will look into adding more developers but are not doing so now.

Reply:
So what are we? We were all added to the project to maintain it and develop it

Reply:
> Alright, I have written up my thoughts in the DLU Developers Group discord and am awaiting thoughts of the rest of the DLU team before I make anymore changes.

There is no "DLU team" as it stands, anyone who was part of the "team" is @'d in this PR. The "DLU team" developed a server and did a FOSS release of it. That team is for the most part no longer active. I very much appreciate the work our contributors have put into this project, but they are not part of any "official DLU team". I'd also appreciate if this can stay mostly on GitHub for transparency.

Reply:
The DLU Developers Group was created to prevent miscommunication as Github is poor as a communication platform, I would much rather be using that for this

Reply:
You are a contributor to an open source project. If I contribute to a random repository, it does not make me part of their organization or internal team. I simply would be an open-source contributor.

Reply:
> The DLU Developers Group was created to prevent miscommunication as Github is poor as a communication platform, I would much rather be using that for this

From what I can tell that discord is invite only, let's keep everything here for transparency.

Reply:
Do you genuinely believe and think that we are nothing more then that?

Reply:
> Do you genuinely believe and think that we are nothing more then that?

I don't really know what you think we owe you. You chose to contribute to an open source project. No one made the promise you'd be on the DLU team. You're a good contributor and deserve credit, but you're not on our internal team.

Reply:
The game is not done there are many features and issues remaining with the server that we are spending literally every day working on and all we wanted was our name at the bottom of the README but that was too much so instead a separate list was made and now we are debating the politics of the naming of that list? We all deserve more then to be considered just contributors and nothing more. We are doing the same work you all did so why are we the lesser?

Reply:
This stance isn't changing. The DLU team are going to be immortalized for delivering the initial open source release and the community gets a separate list for their noteworthy contributors after we delivered the base. You're welcome to disagree and have your own idea about the work that went into this project, but again, DLU team membership is not owed for contributing to this project. You are donating your time and efforts essentially to the public good, as this is a children's MMO emulator that we don't force anyone to work on nor offer incentives. If you're not having a good time or doing this out of interest, don't do it.


If you're unhappy with this, you can close this merge request and I can make the changes to add these names with the proper naming for the DLU team.

Reply:
> The game is not done there are many features and issues remaining with the server that we are spending literally every day working on and all we wanted was our name at the bottom of the README but that was too much so instead a separate list was made and now we are debating the politics of the naming of that list? We all deserve more then to be considered just contributors and nothing more. We are doing the same work you all did so why are we the lesser? One lot of work was done in private, one was done in public.

For all intents and purposes the "DLU team" is dead. It died some months after the FOSS release of DLU when the original team quit maintaining it for the most part and let the FOSS community take over. The "official DLU team" can't accept new members if it's no longer active.

Reply:
> Since my name is on this PR I want to state my thoughts below. yall are too heated at the moment and its not worth it to shout into the void. I asked a simple question yesterday just because i wanted to know if i contributed enough to be put onto the credits at the bottom of the page. I wanted to just put my name at the bottom of it since i had thought I fell under the helm of a Contributor to DLUv3. I got my answer of no and while it is unlucky i will continue to produce the same level of work I do because I find it fun, not because I expect something in return. I just wanted to play one of my favorite games again and learn about production styled code. If I didnt want to do this I just wouldnt do it. I dont waste my time.
> I appreciate jett vouching for me, but its clear the original dlu team is gone and ephemeral.

No one said you aren’t a contributor? We’ve clearly stated you can be listed as a contributor or maintainer.

Reply:
> No one said you aren’t a contributor? We’ve clearly stated you can be listed as a contributor or maintainer.

Yes but my original question was about just tacking it onto the original list.  Clearly there needs to be some distinction from contributors since foss release and post release.  Im fine with that, just ended up being a larger change than I had intended on it being.

Reply:
> > No one said you aren’t a contributor? We’ve clearly stated you can be listed as a contributor or maintainer.
> 
> Yes but my original question was about just tacking it onto the original list. Clearly there needs to be some distinction from contributors since foss release and post release. Im fine with that, just ended up being a larger change than I had intended on it being. I understand having my name on the list is approved but those who got the server to this point want some distinction to show that they got us to a point where we could work on the server. I know my wording here isnt clear or well worded but this is simply because I wasn't ready for this level of discussion on this subject.

I don’t understand what the fuss is about here. We just need @Jettford to change a couple of words, besides that the changes are completely fine. There’s nothing to be gained by arguing here, credit is being given where it is due.

Reply:
I dont think I'm arguing?  Im just discussing what has been said above.  Regardless I've poured enough time into this.  Thanks for adding me to the credits.

Reply:
> I dont think I'm arguing? Im just discussing what has been said above. Regardless I've poured well over enough time into this. Thanks for adding me to the credits.

I wasn’t talking about you, sorry if that was unclear.

Reply:
Resolving thread since the changes were made.